### PR TITLE
Page footer

### PR DESCRIPTION
**Pull request by meyginix**

### CODE REVIEW

File: src/components/Link/Link.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link as GatsbyLink } from 'gatsby'
+
+const isExternalUrl = url => /^(?:(?:https?)|(?:mailto)):/.test(url)
```
Comment (Asen-LoungeLogic) [inline src/components/Link/Link.js]: Should we test for tel as well her?

File: src/components/PageFooter/FooterLogo.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Logo from '../Logo'
+import logo from '../../images/logo-white.svg'
```
Comment (Asen-LoungeLogic) [inline src/components/PageFooter/FooterLogo.js]: Can we have some better naming difference between the component and the asset?

File: src/components/PageFooter/PageFooter.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      { to: '/', caption: 'Home' },
+      { to: '/events', caption: 'Events' },
+      { to: '/jobs', caption: 'Jobs' },
+      { to: '/videos', caption: 'Videos' },
+      { to: '/about', caption: 'About' },
+      { to: 'mailto:info@demuxed.com', caption: 'info@demuxed.com' }
+    ]
+  },
+  {
+    title: 'Legal stuff',
```
Comment (Asen-LoungeLogic) [inline src/components/PageFooter/PageFooter.js]: Lets try and keep track of Titles like that (if this is not per spec)

File: src/components/PageHeader/HeaderNav/HeaderNavItem.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { Link } from 'gatsby'
 
+import Link from '../../Link'
```
Comment (Asen-LoungeLogic) [inline src/components/PageHeader/HeaderNav/HeaderNavItem.js]: Can we have a naming convention for the different type of links (Gatpsy, React etc) wherever possible

File: src/styles/mixins.js [hunk 5]
```diff
@@ -22,7 +22,7 @@ export const fontSize = (size) => {
   const baseLineHeightNum = parseInt(baseLineHeight)
 
   return css`
-    font-size: ${fontSizeNum};
+    font-size: ${fontSizeNum}px;
```
Comment (Asen-LoungeLogic) [inline src/styles/mixins.js]: Should we ensure a cap for this functionality?

Comment (Asen-LoungeLogic) [reply src/styles/mixins.js]: What I mean is we have ${fontSizeNum / baseFontSizeNum} which will be a flat that could go quite long. I believe we cut such numbers to some digit as floats are sometimes misinterpreted by browsers

File: src/components/Link/Link.js (hunk 1 above)
Comment (meyginix) [reply src/components/Link/Link.js]: Done

File: src/components/PageFooter/FooterLogo.js (hunk 2 above)
Comment (meyginix) [reply src/components/PageFooter/FooterLogo.js]: Done

File: src/components/PageFooter/PageFooter.js (hunk 3 above)
Comment (meyginix) [reply src/components/PageFooter/PageFooter.js]: Could you clarify please?

Comment (Asen-LoungeLogic) [reply src/components/PageFooter/PageFooter.js]: I thought this is not what the actual text is supposed to be disregard this cheers!

File: src/styles/mixins.js (hunk 5 above)
Comment (meyginix) [reply src/styles/mixins.js]: Hmm, I see that in production app long float are rounded, but I'm not sure how this is achieved. Looks like Styled Components handles it.

Comment (Asen-LoungeLogic) [reply src/styles/mixins.js]: So I am seeing things like ` line-height: 1.53333;` & `    font-size: 1.33333rem;` could you take a look

Comment (meyginix) [reply src/styles/mixins.js]: Yes, in development these values are `1.5333333333333334` and `1.3333333333333333`. Do we need to round them less than to 5 digits after a point?

Comment (Asen-LoungeLogic) [reply src/styles/mixins.js]: Would it not be best if we cap this to 2 digits after the point?

**Pull request merged**